test/e2e/test_cluster_configs/windows/sac/sac-2004.json

```diff
@@ -0,0 +1,61 @@
+{
+	"env": {
+		"REGION_OPTIONS": "westus2"
```

Comment:
Is there a reason to restrict this to westus2?

Reply:
Not that I know of. It came from the other tests: https://github.com/Azure/aks-engine/tree/master/test/e2e/test_cluster_configs/windows

I will remove that restriction

Reply:
A lot of those tests used resources only available in westus2 (VHD, manage image, shared image gallery). Marketplace images should be available in all regions.